build_tools/benchmarks/generate_cmake_benchmark_suites.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+## Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Generates a CMake file to build the benchmark suites."""
+
+import argparse
+import pathlib
+import suites.cmake_rule_generator
+
+TEMPLATE_DIR = pathlib.Path(__file__).parent
+
+GENERATED_BENCHMARK_SUITES_CMAKE_TEMPLATE = suites.cmake_rule_generator.read_template_from_file(
+    TEMPLATE_DIR, "iree_generated_benchmark_suites_template.cmake")
+
+
+def parse_arguments():
+  """Parses command-line options."""
+
+  parser = argparse.ArgumentParser()
+  parser.add_argument("--output",
```

Comment:
Are we always going to want these in a single file? I guess this is a pretty small wrapper, so it's not hard to change later.